assets/app/scripts/directives/nav.js

```diff
@@ -20,86 +20,78 @@ angular.module('openshiftConsole')
       templateUrl: "views/_sidebar-main-nav-item.html"
     };
   })
-  .directive('projectNav', function($timeout, $location, $filter, LabelFilter) {
+  .directive('projectNav', function($timeout, $location, $filter, LabelFilter, DataService) {
     return {
-      restrict: 'E',
+      restrict: 'EA',
```

Comment:
Well, if we ever decided we wanted valid html, we could then put our directives on `<div>`'s as `data-*` attributes & accomplish the same thing.  I'm guessing we aren't going to worry about that, but figured could leave the option open.